packages/charts/src/nui-charts.less

```
@@ -73,3 +77,151 @@
         break-inside: avoid;
     }
 }
+
+.nui-legend-series {
+    padding: @nui-space-xxs;
+    max-width: @nui-legend-series-max-width;
+    cursor: default; // to prevent text selection cursor
+    .setCssVariable(background-color, nui-color-bg-content);
+
+    &:not(.nui-legend-series--horizontal):not(:last-child) {
+        margin-bottom: 1px;
+    }
+
+    & &__tile-wrapper {
+        display: flex;
+        justify-content: center;
+    }
+
+    & &__tile {
+        display: flex;
+    }
+
+    &--horizontal:not(:last-child) {
+        margin-right: @nui-space-sm;
+    }
+
+    &:empty {
+        display: none;
+    }
+
+    .description-min-width--unset {
```

I think we need an ampersand here since the class is added directly to the nui-legend-series element in dashboards (a.k.a not to a child):

https://github.com/solarwinds/nova/blob/0c3bd4215aa7054e2488b02de9acf19921bd7bde/packages/dashboards/src/lib/components/proportional-widget/proportional-widget.component.html#L48

```suggestion
    &.description-min-width--unset {
```